db/upgrade.php

```diff
@@ -280,5 +280,11 @@ function xmldb_auth_saml2_upgrade($oldversion) {
         upgrade_plugin_savepoint(true, 2019022100, 'auth', 'saml2');
     }
 
+    if ($oldversion < 2019062600) {
+        // Move private key into new setting.
+        set_config('privatekeypass', 'auth_saml2');
```

Comment:
Not sure this looks correct, shouldn't it set the value to get_site_identifier()?

But first lets resolve the other question below, this could be redundant